src/site/stages/build/process-cms-exports/transformers/node-event.js

```diff
@@ -0,0 +1,123 @@
+const {
+  getDrupalValue,
+  utcToEpochTime,
+  getWysiwygString,
+  uriToUrl,
+} = require('./helpers');
+const { mapKeys, camelCase } = require('lodash');
+const assert = require('assert');
+
+function fakeUtc(timeString) {
+  // Assume the timeString looks like 2019-05-30T21:00:00
+  // If it doesn't, we'll need to do some real conversions; fail for now
+  assert(
+    /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}$/.test(timeString),
+    `Expected timeString to look like 2019-05-30T21:00:00, but found ${timeString}`,
+  );
+  return `${timeString.replace('T', ' ')} UTC`;
+}
+
+// Same as the createMetaTag in the transformer helpers, but uses
+// __typename instead of type. Because consistency.
+function createMetaTag(type, key, value) {
+  return {
+    __typename: type,
```

Comment:
:unamused:

Reply:
I ran into the same thing with one of mine that's in progress.  :rage: